packages/im-ott/src/organisms/appSkeleton/AppSkeleton.js

```diff
@@ -1,18 +1,47 @@
-import React from "react";
+import React, { useEffect, useContext, useState } from "react";
+import { Redirect } from "react-router-dom";
 import PropTypes from "prop-types";
+import { useMutation } from "@apollo/client";
 
 // Components
 import { Layout } from "antd";
-import Navbar from "../navbar";
 import FooterContent from "imcomponents/organisms/footerContent";
+import { UserContext } from "imbase/providers/UserProvider";
+import Navbar from "../navbar";
 
 // Styles
 import styles from "./appskeleton.module.scss";
+import { CREATE_USER } from "../../graphql/mutation";
 
 const { Header, Content, Footer } = Layout;
 
 const AppSkeleton = (props) => {
   const { children } = props;
+  const { user } = useContext(UserContext);
+  const [redirect, setredirect] = useState(null);
```

Comment:
why are we setting initial state to null?

Reply:
wherever the user is it will stay because of null
whenever a user is not logged in he will be redirected to the login page